### PR TITLE
Learning Team Agenda - September 17th, 2020

### PR DESCRIPTION
**Pull request by jaredgalanis**
Please add comments or make edits if you want to add to the agenda! Core team and community members are all welcome to participate and add discussion points.

### CODE REVIEW

**Comment by jaredgalanis** [comment]
Let's discuss updating Ember's wikipedia page, see discussion [here](https://discordapp.com/channels/480462759797063690/480777444203429888/755379525667389472).

File: learning-team/2020-09/2020-09-17.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+## Topics
+
+<!-- If you would like to add a topic to the agenda please add a suggestion to the PR using the following format: -->
+<!-- ### Your topic (INITIALS, expected duration in minutes) -->
+<!-- replace with topic -->
+<!-- replace with topic -->
+<!-- replace with topic -->
+<!-- replace with topic -->
+<!-- replace with topic -->
```
Comment (ijlee2) [inline learning-team/2020-09/2020-09-17.md:43]: ### Feedback for Ember Guides (IL, 10 min)

In the Ember Guides, we use the `+` and `-` to indicate what code have been added or deleted. For example, in the [Super Rentals Tutorial](https://guides.emberjs.com/release/tutorial/part-1/orientation/#toc_editing-files-and-live-reload) (and other pages), we might see,

Screen Shot 2020-09-16 at 4 05 14 PM

A developer provided me feedback that 2 interns that they were mentoring weren't sure how to read such code. The developer explained to me that the interns weren't yet familiar with diff tools that we might see on GitHub.

For more complex code (think about a large code review that you might have had to do), I think it's also possible that showing many changes with `+` or `-` may not be the best way to convey the changes.

I wondered if we can **update the code block component** to either:

- Provide a help button that, when clicked, explains how to read code with `+` and `-`
- Provide an option to see Before and After side-by-side (à la [Octane cheat sheet](https://ember-learn.github.io/ember-octane-vs-classic-cheat-sheet/))

Comment (ijlee2) [inline learning-team/2020-09/2020-09-17.md:43]:
```suggestion
### Feedback for Ember Guides (IL, 10 min)

In the Ember Guides, we use the `+` and `-` to indicate what code have been added or deleted. For example, in the [Super Rentals Tutorial](https://guides.emberjs.com/release/tutorial/part-1/orientation/#toc_editing-files-and-live-reload) (and other pages), we might see,

<img width="480" alt="Screen Shot 2020-09-16 at 4 05 14 PM" src="https://user-images.githubusercontent.com/16869656/93392480-7995bc00-f836-11ea-953e-d0161a7538f8.png">

A developer provided me feedback that 2 interns that they were mentoring weren't sure how to read such code. The developer explained to me that the interns weren't yet familiar with diff tools that we might see on GitHub.

For more complex code (think about a large code review that you might have had to do), I think it's also possible that showing many changes with `+` or `-` may not be the best way to convey the changes.

I wondered if we can **update the code block component** to either:

- Provide a help button that, when clicked, explains how to read code with `+` and `-`
- Provide an option to see Before and After side-by-side (à la [Octane cheat sheet](https://ember-learn.github.io/ember-octane-vs-classic-cheat-sheet/))
```

File: learning-team/2020-09/2020-09-17.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+<!-- ### Your topic (INITIALS, expected duration in minutes) -->
+<!-- replace with topic -->
+<!-- replace with topic -->
+<!-- replace with topic -->
+<!-- replace with topic -->
+### Feedback for Ember Guides (IL, 10 min)
+
+In the Ember Guides, we use the `+` and `-` to indicate what code have been added or deleted. For example, in the [Super Rentals Tutorial](https://guides.emberjs.com/release/tutorial/part-1/orientation/#toc_editing-files-and-live-reload) (and other pages), we might see,
+
+<img width="480" alt="Screen Shot 2020-09-16 at 4 05 14 PM" src="https://user-images.githubusercontent.com/16869656/93392480-7995bc00-f836-11ea-953e-d0161a7538f8.png">
```
Comment (ijlee2) [inline learning-team/2020-09/2020-09-17.md:47]:
```suggestion
<img width="480" alt="The Ember Guides uses plus and minus symbols to show code changes" src="https://user-images.githubusercontent.com/16869656/93392480-7995bc00-f836-11ea-953e-d0161a7538f8.png">
```

File: learning-team/2020-09/2020-09-17.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+<!-- replace with topic -->
+<!-- replace with topic -->
+<!-- replace with topic -->
+### Feedback for Ember Guides (IL, 10 min)
+
+In the Ember Guides, we use the `+` and `-` to indicate what code have been added or deleted. For example, in the [Super Rentals Tutorial](https://guides.emberjs.com/release/tutorial/part-1/orientation/#toc_editing-files-and-live-reload) (and other pages), we might see,
+
+<img width="480" alt="The Ember Guides uses plus and minus symbols to show code changes" src="https://user-images.githubusercontent.com/16869656/93392480-7995bc00-f836-11ea-953e-d0161a7538f8.png">
+
+A developer provided me feedback that 2 interns that they were mentoring weren't sure how to read such code. The developer explained to me that the interns weren't yet familiar with diff tools that we might see on GitHub.
```
Comment (ijlee2) [inline learning-team/2020-09/2020-09-17.md:49]:
```suggestion
A developer provided me feedback that 2 interns that they were mentoring weren't sure how to read such code. The developer explained to me that the interns weren't yet familiar with diff tools that we might regularly use on GitHub.
```

File: learning-team/2020-09/2020-09-17.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+- [ ] Jen Weber (JW)
+- [ ] Jessica Jordan (JJ)
+- [x] Kenneth Larsen (KL)
+- [ ] Melanie Sumner (MS)
+- [x] Ricardo Mendes (RM)
+- [ ] Robert Wagner (RW)
+
+### Guests
+- [x] Dan Monroe (DM) 
+- [] Your name here (INITIALS)
```
Comment (ijlee2) [inline learning-team/2020-09/2020-09-17.md:23]:
```suggestion
- [x] James Davis (JD)
```

File: learning-team/2020-09/2020-09-17.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+
+## Topics
+
+<!-- If you would like to add a topic to the agenda please add a suggestion to the PR using the following format: -->
+<!-- ### Your topic (INITIALS, expected duration in minutes) -->
+<!-- replace with topic -->
+<!-- replace with topic -->
+<!-- replace with topic -->
+<!-- replace with topic -->
+
```
Comment (ijlee2) [inline learning-team/2020-09/2020-09-17.md:44]:
```suggestion
```

**Review by ijlee2 — APPROVED** [review]
✨

**Pull request merged**